interface/portraits/portraits_tauren_sprites.gfx

```diff
@@ -72,34 +72,125 @@ spriteTypes = {
 
 	spriteType = {
 		name = "GFX_tauren_male_base_oldage"
-		texturefile = "gfx\\characters\\tauren_male\\tauren_male_base_3.dds"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_base_1.dds"
 		noOfFrames = 1
 		norefcount = yes
 	}
 	
 	spriteType = {
 		name = "GFX_tauren_male_neck"
 		texturefile = "gfx\\characters\\tauren_male\\tauren_male_neck_1.dds"
-		noOfFrames = 12
+		noOfFrames = 24
 		norefcount = yes
 		can_be_lowres = yes
 	}
 
 	spriteType = {
 		name = "GFX_tauren_male_neck_midage"
 		texturefile = "gfx\\characters\\tauren_male\\tauren_male_neck_1.dds"
-		noOfFrames = 12
+		noOfFrames = 24
 		norefcount = yes
 		can_be_lowres = yes
 	}
 
 	spriteType = {
 		name = "GFX_tauren_male_neck_oldage"
-		texturefile = "gfx\\characters\\tauren_male\\tauren_male_neck_3.dds"
-		noOfFrames = 12
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_neck_1.dds"
+		noOfFrames = 24
+		norefcount = yes
+		can_be_lowres = yes
+	}
+	
+	spriteType = {
+		name = "GFX_tauren_male_beard"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_beard_1.dds"
+		noOfFrames = 10
 		norefcount = yes
 		can_be_lowres = yes
 	}
+	
+	spriteType = {
+		name = "GFX_tauren_male_beard_midage"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_beard_1.dds"
+		noOfFrames = 10
+		norefcount = yes
+		can_be_lowres = yes
+	}
+	
+	spriteType = {
+		name = "GFX_tauren_male_beard_oldage"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_beard_1.dds"
+		noOfFrames = 10
+		norefcount = yes
+		can_be_lowres = yes
+	}
+	spriteType = {
+		name = "GFX_tauren_male_hair"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_hair_1.dds"
+		noOfFrames = 8
+		norefcount = yes
+	}	
+	spriteType = {
+		name = "GFX_tauren_male_hair_ties_front"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_hair_ties_front.dds"
+		noOfFrames = 8
+		norefcount = yes
+	}
+	spriteType = {
+		name = "GFX_tauren_male_chin"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_chin_1.dds"
+		noOfFrames = 18
+		norefcount = yes
+		can_be_lowres = yes		
+	}	
+	
+	spriteType = {
+		name = "GFX_tauren_male_cheeks"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_cheeks_1.dds"
+		noOfFrames = 8
+		norefcount = yes
+		can_be_lowres = yes		
+	}
+	
+	spriteType = {
+		name = "GFX_tauren_male_mouth"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_mouth_1.dds"
+		noOfFrames = 18
+		norefcount = yes
+		can_be_lowres = yes		
+	}
+	
+	spriteType = {
+		name = "GFX_tauren_male_clothes"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_clothes.dds"
+		noOfFrames = 12
+		norefcount = yes
+		can_be_lowres = yes		
+	}
+	
+	spriteType = {
+		name = "GFX_tauren_male_nose"
+		texturefile = "gfx\\characters\\tauren_male\\tauren_male_nose_1.dds"
+		noOfFrames = 8
+		norefcount = yes
```

Comment:
Also, resave all the spites without generating mipmaps. They're not needed and increase file size.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105495982-4a0c2a00-5cd6-11eb-8b9c-d33283624ddb.png)

</details>